src/_guides/language/effective-dart/design.md

```diff
@@ -1183,7 +1168,7 @@ List<List<Ingredient>> possibleDesserts(Set<Ingredient> pantry) {
 {% endprettify %}
 
 If the local variable doesn't have an initializer, then its type can't be
-inferred. In that case, it *is* a good idea to annotate it. Otherwise, you get
+inferred. In that case, it *is* a good idea to annotate. Otherwise, you get
 `dynamic` and lose the benefits of types.
```

Comment:
Consider:

lose the benefits of types --> lose the benefits of static type checking

Reply:
Done.